l1-contracts/src/UniFiAVSManager.sol

```diff
@@ -32,6 +34,11 @@ contract UniFiAVSManager is UniFiAVSManagerStorage, IUniFiAVSManager, UUPSUpgrad
      * @custom:oz-upgrades-unsafe-allow state-variable-immutable
      */
     IDelegationManager public immutable override EIGEN_DELEGATION_MANAGER;
+    /**
+     * @notice The RewardsCoordinator contract
+     * @custom:oz-upgrades-unsafe-allow state-variable-immutable
```

Comment:
We can remove these `* @custom:oz-upgrades-unsafe-allow state-variable-immutable` they are useless, since we aren't really using the plugin how it is supposed to be used :D